contrail-controller/hooks/contrail_controller_utils.py

```diff
@@ -179,3 +185,66 @@ def update_charm_status():
         docker_utils.compose_run(REDIS_CONFIGS_PATH + "/docker-compose.yaml")
 
     common_utils.update_services_status(SERVICES)
+
+
+def update_hosts_file(hosts_map):
```

Comment:
i see that all client pass just one item to this function - maybe it's better to change interface from host_map to (ip, hostname) ?

Reply:
Makes sense, will do.